rucio-server/templates/auth_deployment.yaml

```diff
@@ -157,8 +166,6 @@ spec:
               value: "{{ .Values.serverType.authServer }}"
             - name: RUCIO_LOG_FORMAT
               value: '{{ .Values.logFormat.authServer }}'
-          resources:
-{{ toYaml .Values.resources | indent 12 }}
```

Comment:
Why don't you just change this to `{{ toYaml .Values.authServerResources | indent 12 }}` and then add `authServerResources: {}` to the default values.yaml? This would keep the template a bit simpler and you would directly get some examples from values.yaml.